assets/i18n/cy.json

```diff
@@ -400,5 +401,11 @@
   "NO_AWARD_NO_LONGER_RECEIVING_BENEFIT": "nid ydych yn cael Budd-dal Plant mwyach",
   "NO_AWARD_YOU_CAN": "Gallwch",
   "NO_AWARD_FIND_OUT_MORE_LINKTEXT": "ddysgu rhagor am Fudd-dal Plant",
-  "LOADING": "Llwytho"
+  "LOADING": "Llwytho",
+  "COB_PAGE_HEADING": "Change the  account you use for Child Benefit payments",
```

Comment:
May be an extra space after keyword '  account'

![image](https://github.com/user-attachments/assets/bca46545-ac54-4858-a4da-42fb75f1e70a)